variables.sh

```diff
@@ -1,13 +1,20 @@
 #!/bin/bash -e
 
+BUILDS=("protoc-all" "protoc" "prototool" "grpc-cli" "gen-grpc-gateway")
 DOCKER_REPO=${DOCKER_REPO}
 NAMESPACE=${NAMESPACE:-namely}
-GRPC_VERSION=${GRPC_VERSION:-1.34}
-GRPC_JAVA_VERSION=${GRPC_JAVA_VERSION:-1.34}
-GRPC_WEB_VERSION=${GRPC_WEB_VERSION:-1.2.1}
-GRPC_GATEWAY_VERSION=${GRPC_GATEWAY_VERSION:-2.0.1}
-GO_VERSION=${GO_VERSION:-1.14}
-BUILD_VERSION=${BUILD_VERSION:-5}
+BUILD_VERSION=${BUILD_VERSION:-6}
 CONTAINER=${DOCKER_REPO}${NAMESPACE}
 LATEST=${1:false}
-BUILDS=("protoc-all" "protoc" "prototool" "grpc-cli" "gen-grpc-gateway")
+
+GO_VERSION=${GO_VERSION:-1.14}
+GRPC_VERSION=${GRPC_VERSION:-1.36}
+GRPC_JAVA_VERSION=${GRPC_JAVA_VERSION:-1.36}
+GRPC_WEB_VERSION=${GRPC_WEB_VERSION:-1.2.1}
+GRPC_GATEWAY_VERSION=${GRPC_GATEWAY_VERSION:-2.0.1}
+UBER_PROTOTOOL_VERSION=${UBER_PROTOTOOL_VERSION:-1.3.0}
+SCALA_PB_VERSION=${SCALA_PB_VERSION:-0.11.0}
+NODE_VERSION=${NODE_VERSION:-15}
```

Comment:
if there's any way we could stay on the latest LTS of Node, I'd appreciate it.  Today that's 14.x latest.